n3fit/src/n3fit/layers/preprocessing.py

```diff
@@ -31,28 +33,23 @@ class Preprocessing(MetaLayer):
                             (defaults to true)
         large_x: bool
             Whether large x preprocessing factor should be active
-        seed: int
-            seed for the initializer of the random alpha and beta values
-        num_replicas: int (default 1)
-            The number of replicas
     """
 
     def __init__(
         self,
+        replica_seeds: Optional[List[int]],
```

Comment:
```suggestion
        replica_seeds: List[int],
```

otherwise add a default